contrib/epee/include/net/abstract_tcp_server.h

```diff
@@ -38,6 +38,9 @@
 
 #pragma comment(lib, "Ws2_32.lib")
 
+#undef MONERO_DEFAULT_LOG_CATEGORY
+#define MONERO_DEFAULT_LOG_CATEGORY "net"
```

Comment:
My only thought would be to explicitly state the category for each log call. It doesn't seem that "wordy" to me, but outputting the wrong category isn't that terrible either.